### PR TITLE
api: add eds config to csds proto

### PR DESCRIPTION
**Pull request by Rainton**
Commit Message:
add EDS config to csds proto.

Additional Description:
EDS config has been added to [config_dump](https://github.com/envoyproxy/envoy/blob/master/api/envoy/admin/v3/config_dump.proto#L355) in #11425  and implemented in #11577 .

Risk Level: Low
Testing: N/A
Docs Changes: N/A
Release Notes: N/A

Signed-off-by: Yutong Li <yutongli@google.com>
/cc @fuqianggao @alexburnos

### CODE REVIEW

File: api/envoy/service/status/v3/csds.proto [hunk 1]
```diff
@@ -79,6 +79,8 @@ message PerXdsConfig {
     admin.v3.RoutesConfigDump route_config = 4;
 
     admin.v3.ScopedRoutesConfigDump scoped_route_config = 5;
+
+    admin.v3.EndpointsConfigDump endpoint_config = 6;
```
Comment (htuch) [inline api/envoy/service/status/v3/csds.proto:83]: Should this be `[#not-implemented-yet:]`?

Comment (Rainton) [reply api/envoy/service/status/v3/csds.proto:84]: Sorry about that, solved

**Comment by Rainton** [comment]
@htuch Sorry about my misspelling. Just corrected it to `[#not-implemented-hide:]`

**Comment by fuqianggao** [comment]
LGTM. Thank you.

**Pull request merged**